### PR TITLE
Make Certify block non-editable if non-staff user (AGM Filings)

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#18161

*Description of changes:*
- Certify block is now not editable for non-staff users (AGM filings)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/views/AgmLocationChg.vue [hunk 1]
```diff
@@ -419,7 +420,7 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
   mounted (): void {
     // Pre-populate the certified block with the logged in user's name (if not staff)
     if (!this.isRoleStaff && this.getUserInfo) {
-      this.certifiedBy = this.getUserInfo.firstname + ' ' + this.getUserInfo.lastName
+      this.certifiedBy = this.getUserInfo.firstname + ' ' + this.getUserInfo.lastname
```
Comment (severinbeauvais) [inline src/views/AgmLocationChg.vue:423]: So sorry about this 😊 Thanks for fixing it.

**Review by severinbeauvais — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-564-dhq8toxr.web.app

**Review by ketaki-deodhar — APPROVED** [review]
Looks good 👍

File: src/views/AgmLocationChg.vue (hunk 1 above)
Comment (JazzarKarim) [reply src/views/AgmLocationChg.vue:423]: No worries Sev! Happened to me too when I first wrote the variables. I had `firstName` and `lastName` so I was getting undefined undefined 😆

**Pull request merged**